css-three-column-layout/index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Three Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <div class="row">
+      <div class="col-full center pad2">
+        <h1>Designed for life science. Optimized for SaMD and Digital Health.</h1>
+      </div>
+    </div>
+    <div class="row center">
+      <div class="col-third resize bg margin">
+        <img class="padhalf" src="images/merge.png">
+      </div>
+      <div class="col-third resize bg margin">
+        <img class="padhalf" src="images/cloud-computing.png">
+      </div>
+      <div class="col-third resize bg margin">
+        <img class="padhalf" src="images/discussion.png">
+      </div>
+    </div>
+    <div class="row center">
+      <div class="col-third resize margin">
+        <section>
+          <h2>Streamlined QMS</h2>
+          <p>
+            A QMS that includes modules for all stages of the product life cycle from Design Control to CAPA
+          </p>
+        </section>
+      </div>
+      <div class="col-third resize margin">
+        <section>
+          <h2>Integrates with existing tools</h2>
+          <p>
+            Enzyme QMS integrates with your existing tools, such as JIRA, GitHub and Jenkins with Enzyme.
+          </p>
+        </section>
+      </div>
+      <div class="col-third resize margin">
+        <section>
+          <h2>In-depth Onboarding</h2>
+          <p>
+            We train your entire team and provide you with support when and as it's needed
+          </p>
+        </section>
+      </div>
+    </div>
+  </div>
```

Comment:
I can see why you would do this, but the images and text are related to each other, so they shouldn't be in separate rows. You should have three `col-third` elements, each of which containing the image, headline, and text for each item.